commit 423ae3bd04bb63dd111c0afc2a888e5f3e375913
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Jul 27 16:00:57 2021 -0300

    Obl ou obj: lista de verbos seguidos de "sobre", "entre" e outros

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b1ba9f5..a903956 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -4938,7 +4938,7 @@
 5	sobre	sobre	ADP	_	_	8	case	O	_
 6	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 7	antiga	antigo	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
-8	zona	zona	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
+8	zona	zona	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	fraqueza	fraqueza	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 11	,	,	PUNCT	_	_	14	punct	O	_
@@ -6295,7 +6295,7 @@
 7	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 8	sob	sob	ADP	_	_	10	case	O	_
 9	espesso	espesso	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
-10	prisma	prisma	NOUN	_	Gender=Masc|Number=Sing	7	obj	O	_
+10	prisma	prisma	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 11	sedimentar	sedimentar	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
 12	devido	dever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	15	case	O	MWEPOS=ADP
 13-14	ao	_	_	_	_	_	_	_	_
@@ -7704,7 +7704,7 @@
 11	vai	ir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 12	desde	desde	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	cidade	cidade	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
+14	cidade	cidade	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	Itajaí	Itajaí	PROPN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
@@ -12071,7 +12071,7 @@
 11	variar	variar	VERB	_	VerbForm=Inf	10	xcomp	O	_
 12	entre	entre	ADP	_	_	14	case	O	_
 13	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
-14	líquido	líquido	NOUN	_	Gender=Masc|Number=Sing	11	obj	O	_
+14	líquido	líquido	NOUN	_	Gender=Masc|Number=Sing	11	obl	O	_
 15	de	de	ADP	_	_	17	case	O	_
 16	baixa	baixo	ADJ	_	Gender=Fem|Number=Sing	17	amod	O	_
 17	consistência	consistência	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
@@ -16999,7 +16999,7 @@
 28	catiônicas	catiônico	ADJ	_	Gender=Fem|Number=Plur	26	amod	O	_
 29	aplicadas	aplicar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	26	acl	O	_
 30	sobre	sobre	ADP	_	_	31	case	O	_
-31	agregados	agregado	NOUN	_	Gender=Masc|Number=Plur	29	obl	O	_
+31	agregados	agregado	NOUN	_	Gender=Masc|Number=Plur	29	obj	O	_
 32	graúdos	graúdo	ADJ	_	Gender=Masc|Number=Plur	31	amod	O	_
 33	.	.	PUNCT	_	_	3	punct	O	_
 34	.	.	PUNCT	_	_	3	punct	O	_
@@ -18624,7 +18624,7 @@
 12	ficou	ficar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 13	entre	entre	ADP	_	_	15	case	O	_
 14	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	15	det	O	_
-15	padrões	padrão	NOUN	_	Gender=Masc|Number=Plur	12	obj	O	_
+15	padrões	padrão	NOUN	_	Gender=Masc|Number=Plur	12	obl	O	_
 16	estabelecidos	estabelecer	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	15	acl	O	_
 17-18	pela	_	_	_	_	_	_	_	_
 17	por	por	ADP	_	_	19	case	O	_
@@ -22356,11 +22356,11 @@
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
 16	Nordeste	nordeste	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
-18	a	ela	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	12	conj	O	_
+18	a	ela	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	9	conj	O	_
 19	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	20	nsubj	O	_
 20	apresenta	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	18	acl:relcl	O	_
-21	,	,	PUNCT	_	_	20	punct	O	_
-22	dentre	dentre	ADP	_	_	25	case	O	_
+21	,	,	PUNCT	_	_	23	punct	O	_
+22	dentre	dentre	ADP	_	_	23	case	O	_
 23	todas	todo	PRON	_	Gender=Fem|Number=Plur|PronType=Tot	20	obl	O	_
 24	,	,	PUNCT	_	_	23	punct	O	_
 25	história	história	NOUN	_	Gender=Fem|Number=Sing	20	obj	O	_
@@ -22368,7 +22368,7 @@
 27	mais	mais	ADV	_	_	28	advmod	O	_
 28	complexa	complexo	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
 29	e	e	CCONJ	_	_	30	cc	O	_
-30	empilhamento	empilhamento	NOUN	_	Gender=Masc|Number=Sing	28	conj	O	_
+30	empilhamento	empilhamento	NOUN	_	Gender=Masc|Number=Sing	25	conj	O	_
 31	estratigráfico	estratigráfico	ADJ	_	Gender=Masc|Number=Sing	30	amod	O	_
 32	mais	mais	ADV	_	_	33	advmod	O	_
 33	completo	completo	ADJ	_	Gender=Masc|Number=Sing	30	amod	O	_
@@ -22393,13 +22393,13 @@
 50	Província	Província	PROPN	_	Number=Sing	46	flat:name	O	_
 51	Borborema	Borborema	PROPN	_	Number=Sing	46	flat:name	O	_
 52	(	(	PUNCT	_	_	53	punct	O	_
-53	Brito	Brito	PROPN	_	Gender=Masc|Number=Sing	46	appos	O	_
+53	Brito	Brito	PROPN	_	Gender=Masc|Number=Sing	40	appos	O	_
 54	Neves	Neves	PROPN	_	Number=Sing	53	flat:name	O	_
 55	et	et	PROPN	_	Number=Sing	53	flat:name	O	_
 56	al.	al.	PROPN	_	Number=Sing	53	flat:name	O	_
 57	2000	2000	PROPN	_	Number=Sing	53	flat:name	O	_
 58	)	)	PUNCT	_	_	53	punct	O	_
-59	,	,	PUNCT	_	_	40	punct	O	_
+59	,	,	PUNCT	_	_	61	punct	O	_
 60	a	a	ADP	_	_	61	case	O	_
 61	sul	sul	NOUN	_	Gender=Masc|Number=Sing	40	obl	O	_
 62-63	do	_	_	_	_	_	_	_	_
@@ -23571,7 +23571,7 @@
 12	quase	quase	ADV	_	_	8	advmod	O	_
 13	todo	todo	DET	_	Gender=Masc|Number=Sing|PronType=Tot	15	det	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
-15	sopé	sopé	NOUN	_	Gender=Masc|Number=Sing	8	obj	O	_
+15	sopé	sopé	NOUN	_	Gender=Masc|Number=Sing	8	obl	O	_
 16-17	da	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	18	case	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
@@ -149340,7 +149340,7 @@
 9	compreendido	compreender	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	7	acl	O	_
 10	entre	entre	ADP	_	_	12	case	O	_
 11	130	130	NUM	_	NumType=Card	12	nummod	O	_
-12	Ma	ma	NOUN	_	Gender=Masc|Number=Plur	9	obl	O	_
+12	Ma	ma	NOUN	_	Gender=Masc|Number=Plur	9	obj	O	_
 13	e	e	CCONJ	_	_	15	cc	O	_
 14	125	125	NUM	_	NumType=Card	15	nummod	O	_
 15	Ma	Ma	PROPN	_	Gender=Masc|Number=Plur	12	conj	O	_
@@ -150898,7 +150898,7 @@
 14	compreendido	compreender	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	12	acl	O	_
 15	entre	entre	ADP	_	_	17	case	O	_
 16	2.5	2.5	NUM	_	NumType=Card	17	nummod	O	_
-17	Ma	ma	NOUN	_	Gender=Masc|Number=Plur	14	obl	O	_
+17	Ma	ma	NOUN	_	Gender=Masc|Number=Plur	14	obj	O	_
 18	e	e	CCONJ	_	_	20	cc	O	_
 19	0.0117	0.0117	NUM	_	NumType=Card	20	nummod	O	_
 20	Ma	Ma	PROPN	_	Gender=Masc|Number=Sing	17	conj	O	_
@@ -162593,7 +162593,7 @@
 10	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 11	sobre	sobre	ADP	_	_	13	case	O	_
 12	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
-13	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	10	obj	O	_
+13	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	10	obl	O	_
 14	pouco	pouco	ADV	_	_	15	advmod	O	_
 15	perturbada	perturbar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	13	acl	O	_
 16	.	.	PUNCT	_	_	10	punct	O	_
@@ -170236,7 +170236,7 @@
 12	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	11	expl	O	_
 13	sobre	sobre	ADP	_	_	15	case	O	_
 14	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
-15	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
+15	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 16	sólida	sólido	ADJ	_	Gender=Fem|Number=Sing	15	amod	O	_
 17-18	na	_	_	_	_	_	_	_	_
 17	em	em	ADP	_	_	19	case	O	_
@@ -180797,7 +180797,7 @@
 28	seguinte	seguinte	NOUN	_	Gender=Masc|Number=Sing	26	obj	O	_
 29	sobre	sobre	ADP	_	_	31	case	O	_
 30	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	31	det	O	_
-31	efeitos	efeito	NOUN	_	Gender=Masc|Number=Plur	26	obl	O	_
+31	efeitos	efeito	NOUN	_	Gender=Masc|Number=Plur	26	obj	O	_
 32-33	do	_	_	_	_	_	_	_	_
 32	de	de	ADP	_	_	34	case	O	_
 33	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	det	O	_
@@ -188024,7 +188024,7 @@
 3	divergem	divergir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	sobre	sobre	ADP	_	_	6	case	O	_
 5	qual	qual	DET	_	Gender=Masc|Number=Sing|PronType=Int	6	det	O	_
-6	combustível	combustível	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
+6	combustível	combustível	NOUN	_	Gender=Masc|Number=Sing	3	obj	O	_
 7	emite	emitir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	ccomp	O	_
 8	maior	grande	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
 9	quantidade	quantidade	NOUN	_	Gender=Fem|Number=Sing	7	obj	O	_
@@ -192448,7 +192448,7 @@
 9	acontece	acontecer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	ccomp	O	_
 10	sem	sem	ADP	_	_	12	case	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	presença	presença	NOUN	_	Gender=Fem|Number=Sing	9	obj	O	_
+12	presença	presença	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
 13	de	de	ADP	_	_	15	case	O	_
 14	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
 15	catalisador	catalisador	NOUN	_	Gender=Masc|Number=Sing	12	nmod	B=QUÍMICA	_
@@ -212300,7 +212300,7 @@
 22	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	23	nsubj	O	_
 23	incide	incidir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	20	acl:relcl	O	_
 24	sobre	sobre	ADP	_	_	25	case	O	_
-25	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	23	obj	O	_
+25	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	23	obl	O	_
 26	,	,	PUNCT	_	_	27	punct	O	_
 27	deixando	deixar	VERB	_	VerbForm=Ger	23	advcl	O	_
 28	transmitir	transmitir	VERB	_	VerbForm=Inf	27	xcomp	O	_
@@ -231331,7 +231331,7 @@
 29	publicados	publicar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	28	acl	O	_
 30	sobre	sobre	ADP	_	_	32	case	O	_
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
-32	área	área	NOUN	_	Gender=Fem|Number=Sing	29	obl	O	_
+32	área	área	NOUN	_	Gender=Fem|Number=Sing	29	obj	O	_
 33	(	(	PUNCT	_	_	34	punct	O	_
 34	Zalán	Zalán	PROPN	_	Gender=Masc|Number=Sing	5	parataxis	O	_
 35	e	e	CCONJ	_	_	34	flat:name	O	_
@@ -243171,7 +243171,7 @@
 6	técnicos	técnico	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
 7	sobre	sobre	ADP	_	_	9	case	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
-9	aquisição	aquisição	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
+9	aquisição	aquisição	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
 10-11	destes	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
 11	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	12	det	O	_
@@ -243181,15 +243181,15 @@
 14	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 15	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	14	expl	O	_
 16	dizer	dizer	VERB	_	VerbForm=Inf	14	xcomp	O	_
-17	,	,	PUNCT	_	_	16	punct	O	_
+17	,	,	PUNCT	_	_	18	punct	O	_
 18	independentemente	independentemente	ADV	_	_	16	advmod	O	_
 19	de	de	ADP	_	_	21	case	O	_
 20	qualquer	qualquer	DET	_	Gender=Fem|Number=Sing|PronType=Ind	21	det	O	_
 21	análise	análise	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
-22	,	,	PUNCT	_	_	16	punct	O	_
-23	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	25	nsubj	O	_
+22	,	,	PUNCT	_	_	18	punct	O	_
+23	que	que	SCONJ	_	_	25	mark	O	_
 24	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	25	expl	O	_
-25	trata	tratar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	21	acl:relcl	O	_
+25	trata	tratar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	16	xcomp	O	_
 26	de	de	ADP	_	_	28	case	O	_
 27	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
 28	conjunto	conjunto	NOUN	_	Gender=Masc|Number=Sing	25	obj	O	_
@@ -243202,7 +243202,7 @@
 35	extremamente	extremamente	ADV	_	_	36	advmod	O	_
 36	confiável	confiável	ADJ	_	Gender=Masc|Number=Sing	28	amod	O	_
 37	,	,	PUNCT	_	_	38	punct	O	_
-38	obtido	obter	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	36	conj	O	_
+38	obtido	obter	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	28	acl:relcl	O	_
 39	dentro	dentro	ADV	_	_	38	advmod	O	_
 40	de	de	ADP	_	_	42	case	O	_
 41	rigoroso	rigoroso	ADJ	_	Gender=Masc|Number=Sing	42	amod	O	_
@@ -243644,7 +243644,7 @@
 3	compreendida	compreender	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	2	acl	O	_
 4	entre	entre	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
-6	Guanabara	Guanabara	PROPN	_	Gender=Fem|Number=Sing	3	obl	O	_
+6	Guanabara	Guanabara	PROPN	_	Gender=Fem|Number=Sing	3	obj	O	_
 7	e	e	CCONJ	_	_	8	cc	O	_
 8	Cabo	Cabo	PROPN	_	Gender=Masc|Number=Sing	6	conj	O	_
 9	Frio	Frio	PROPN	_	Number=Sing	8	flat:name	O	_
@@ -249890,7 +249890,7 @@
 24	,	,	PUNCT	_	_	10	punct	O	_
 25	sobre	sobre	ADP	_	_	27	case	O	_
 26	as	o	DET	_	Gender=Fem|Number=Plur|PronType=Art	27	det	O	_
-27	quais	qual	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	29	obl	O	_
+27	quais	qual	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	29	obj	O	_
 28	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	29	aux:pass	O	_
 29	aplicada	aplicar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	22	acl:relcl	O	_
 30	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	31	det	O	_
@@ -250986,7 +250986,7 @@
 14	próximas	próximo	ADJ	_	Gender=Fem|Number=Plur	13	amod	O	_
 15	interagindo	interagir	VERB	_	VerbForm=Ger	11	advcl	O	_
 16	entre	entre	ADP	_	_	17	case	O	_
-17	si	si	PRON	_	Gender=Masc|Number=Sing|Person=3|PronType=Prs	15	obl	O	_
+17	si	si	PRON	_	Gender=Masc|Number=Sing|Person=3|PronType=Prs	15	obj	O	_
 18	ou	ou	CCONJ	_	_	19	cc	O	_
 19	corpos	corpo	NOUN	_	Gender=Masc|Number=Plur	17	conj	O	_
 20	contrastantes	contrastante	ADJ	_	Gender=Masc|Number=Plur	19	amod	O	_
@@ -273766,7 +273766,7 @@
 9	concentrou	concentrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 10	sobre	sobre	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
-12	altos	alto	NOUN	_	Gender=Masc|Number=Plur	9	obj	O	_
+12	altos	alto	NOUN	_	Gender=Masc|Number=Plur	9	obl	O	_
 13	estruturais	estrutural	ADJ	_	Gender=Masc|Number=Plur	12	amod	O	_
 14-15	deste	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	16	case	O	_
@@ -275906,7 +275906,7 @@
 21-22	dos	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	20	fixed	O	_
 22	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
-23	limites	limite	NOUN	_	Gender=Masc|Number=Plur	19	obj	O	_
+23	limites	limite	NOUN	_	Gender=Masc|Number=Plur	19	obl	O	_
 24-25	da	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
@@ -283421,7 +283421,7 @@
 22	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	19	acl:relcl	O	_
 23	sob	sob	ADP	_	_	25	case	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
-25	forma	forma	NOUN	_	Gender=Fem|Number=Sing	22	obj	O	_
+25	forma	forma	NOUN	_	Gender=Fem|Number=Sing	22	obl	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	diques..	diques..	NOUN	_	Gender=Masc|Number=Sing	25	nmod	O	_
 
@@ -295950,11 +295950,11 @@
 11	gráfico	gráfico	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
 12	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	13	nsubj	O	_
 13	mostra	mostrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	acl:relcl	O	_
-14	como	como	ADV	_	_	16	case	O	_
+14	como	como	SCONJ	_	_	18	mark	O	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
-16	tensão	tensão	NOUN	_	Gender=Fem|Number=Sing	13	obj	O	_
+16	tensão	tensão	NOUN	_	Gender=Fem|Number=Sing	18	nsubj	O	_
 17	cisalhante	cisalhante	ADJ	_	Gender=Fem|Number=Sing	16	amod	O	_
-18	varia	variar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	acl:relcl	O	_
+18	varia	variar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	ccomp	O	_
 19	em	em	ADP	_	_	20	case	O	_
 20	função	função	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
 21-22	da	_	_	_	_	_	_	_	_
@@ -320870,7 +320870,7 @@
 11	flui	fluir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 12	até	até	ADP	_	_	14	case	O	_
 13	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	tensão	tensão	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
+14	tensão	tensão	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 17	,	,	PUNCT	_	_	18	punct	O	_
@@ -334392,7 +334392,7 @@
 13	atuando	atuar	VERB	_	VerbForm=Ger	8	advcl	O	_
 14	sobre	sobre	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
-16	produto	produto	NOUN	_	Gender=Masc|Number=Sing	13	obj	O	_
+16	produto	produto	NOUN	_	Gender=Masc|Number=Sing	13	obl	O	_
 17	já	já	ADV	_	_	18	advmod	O	_
 18	envelhecido	envelhecido	ADJ	_	Gender=Masc|Number=Sing	16	amod	O	_
 19	.	.	PUNCT	_	_	1	punct	O	_
